src/number_theory/modular.lean

```diff
@@ -17,6 +17,25 @@ We define the action of `SL(2,ℤ)` on `ℍ` (via restriction of the `SL(2,ℝ)`
 (`modular_group.exists_smul_mem_fundamental_domain`) that any point in `ℍ` can be
 moved inside `𝒟`.
 
+## Main definitions
+
+The standard (closed) fundamental domain of the action of `SL(2,ℤ)` on `ℍ`:
+`fundamental_domain := {z | 1 ≤ (z : ℂ).norm_sq ∧ |z.re| ≤ (1 : ℝ) / 2}`
+
+The standard open fundamental domain of the action of `SL(2,ℤ)` on `ℍ`:
+`fundamental_domain_open := {z | 1 < (z : ℂ).norm_sq ∧ |z.re| < (1 : ℝ) / 2}`
+
+
+## Main results
+
+Any `z : ℍ` can be moved to `𝒟` by an element of `SL(2,ℤ)`:
+`exists_smul_mem_fundamental_domain (z : ℍ) : ∃ g : SL(2,ℤ), g • z ∈ 𝒟`
+
+If both `z` and `γ • z` are in the open domain `𝒟ᵒ` then `z = γ • z`:
+`eq_smul_self_of_mem_fdo_mem_fdo {z : ℍ} {g : SL(2,ℤ)} (hz : z ∈ 𝒟ᵒ) (hg : g • z ∈ 𝒟ᵒ) : z = g • z`
+
+# Discussion
```

Comment:
Can you add somewhere a suggestion to open the `modular` locale to use these results? Thanks!